src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -341,6 +345,14 @@ public function reset_tracking_options() {
 	}
 
 	public function continue_import() {
+		global $wpdb;
+
+		$lock_key = 'tribe_ea_csv_import_' . $this->id;
+
+		if ( empty( $this->acquire_db_lock( $lock_key ) ) ) {
```

Comment:
Since `acquire_db_lock` always returns a boolean I think the empty can be removed. 
```suggestion
		if ( $this->acquire_db_lock( $lock_key ) ) {
```